multihost_test/bz_automation/test_automation.py

```diff
@@ -254,3 +254,20 @@ def test_21244(self, multihost, create_localusers, bkp_pam_config):
         with pytest.raises(Exception):
             client.run_command("sh /tmp/2014458.sh")
         client.run_command("rm -vf /tmp/2014458.sh")
+
+    def test_5783(self, multihost, create_localusers, bkp_pam_config):
+        """
+        :title: pam_tty_audit.so: fails with error "cannot open session: Permission denied"
+        :id: c9dbce70-2d4d-11ef-b036-845cf3eff344
+        :bugzilla: https://issues.redhat.com/browse/SSSD-5783
+        :steps:
+            1. Edit /etc/pam.d/system-auth file enable pam_tty_audit.so with options "disable=* enable=<user list>". Make
+                the user list under "enable" is long. Making the total string of 1024 or more.
```

Comment:
Edit /etc/pam.d/system-auth file to add pam_tty_audit.so with options "disable=* enable=<user list>". Make the user list under "enable" longer than 1024 characters.